gradle.properties

```diff
@@ -1 +1,2 @@
+substrateVersion = 0.0.4-SNAPSHOT
```

Comment:
Rename the property to `version`. You can then remove it altogether from the `build.gradle` of the substrate project.

Reply:
done